CRM/Financial/Form/FrontEndPaymentFormTrait.php

```diff
@@ -43,25 +43,39 @@ trait CRM_Financial_Form_FrontEndPaymentFormTrait {
    * CRM_Invoicing_Utils class with a potential end goal of moving this handling to an extension.
    *
    * @param $tplLineItems
-   *
-   * @return array
    */
-  protected function alterLineItemsForTemplate($tplLineItems) {
-    $getTaxDetails = FALSE;
+  protected function alterLineItemsForTemplate(&$tplLineItems) {
+    if (CRM_Invoicing_Utils::isInvoicingEnabled()) {
```

Comment:
@eileenmcnaughton should there be ! before the CRM...?

Reply:
opps - fixed